server/ntb/tests/io/feed_parsers/tt_newsml_test.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from ntb.io.feed_parsers.tt_newsml import TTNewsMLFeedParser
+from . import XMLParserTestCase
+
+
+class BaseTTNewsMLTestCase(XMLParserTestCase):
+    def setUp(self):
+        self.parser = TTNewsMLFeedParser()
+        super().setUp()
+
+
+class TTTestCase(BaseTTNewsMLTestCase):
+    filename = 'tt_newsml_test.xml'
+
+    def test_can_parse(self):
+        self.assertTrue(TTNewsMLFeedParser().can_parse(self.xml_root))
+
+    def test_content(self):
+        item = self.item[0]
+        self.assertEqual(item['subject'], [{'qcode': 'Utenriks', 'name': 'Utenriks', 'scheme': 'category'}])
+        self.assertEqual(item['anpa_category'], [{'name': 'Nyhetstjenesten', 'qcode': 'n'}])
+        self.assertEqual(item['headline'], 'Polen: EU hämnas för flyktingmotstånd')
+        self.assertEqual(item['guid'], 'urn:newsml:tt.se:20171220:TT:585167:0')
+        self.assertEqual(item['uri'], 'urn:newsml:tt.se:20171220:TT:585167:0')
+        self.assertEqual(item['firstcreated'].isoformat(), '2017-12-20T14:46:21+01:00')
+        self.assertEqual(item['abstract'], 'För första gången sedan EU:s Lissabonfördrag trädde i kraft tar EU-kommiss'
+                                           'ionen i med sin storslägga mot ett medlemsland. Polen riskerar nu formellt'
+                                           ' indragen rösträtt - även om vägen dit fortfarande är lång.\n            '
+                                           '\nI Warszawa tror man att åtgärden är en hämnd för landets ovilja att ta e'
+                                           'mot flyktingar.\n          ')
+
+        expected_body = ('<div><h1>Polen: EU hämnas för flyktingmotstånd</h1>\n          <p>EU / Bryssel, TT:s korresp'
```

Comment:
can we avoid that `div` around the whole body? that should allow the editor to split it into blocks

Reply:
OK